### PR TITLE
Created ChatScreen with AppBar and NavigationBar

### PR DESCRIPTION
**Pull request by DevHasan7**
NavigationBar kopiert von Jonas

### CODE REVIEW

**Comment by jonas2808** [comment]
@DevHasan7 I've updated your PR with current master together with @Vatsegkan in a call to show him Github a bit.

File: sth_app/lib/main.dart [hunk 1]
```diff
@@ -9,17 +10,9 @@ class SthApp extends StatelessWidget {
...
-      routes: {
-        '/homescreen': (context) => const HomeScreen(),
-        // Add more routes as needed
-        // Example:
-        // '/secondPage': (context) => SecondPage(),
-      },
-    );
+    return MaterialApp(title: 'STH App', debugShowCheckedModeBanner: false, initialRoute: '/homescreen', routes: {
+      '/homescreen': (context) => const HomeScreen(),
+      '/chatScreen': (context) => const chatScreen(),
```
Comment (jonas2808) [inline sth_app/lib/main.dart:15]: Can we write '/chatscreen': (context) => const ChatScreen(), ??

File: sth_app/lib/pages/chatscreen/chatscreen.dart [hunk 2]
```diff
@@ -1 +1,48 @@
+import 'package:flutter/material.dart';
 
+class chatScreen extends StatefulWidget {
```
Comment (jonas2808) [inline sth_app/lib/pages/chatscreen/chatscreen.dart:3]: ChatScreen

File: sth_app/lib/pages/chatscreen/chatscreen.dart [hunk 3]
```diff
@@ -1 +1,48 @@
...
+  const chatScreen({Key? key}) : super(key: key);
```
Comment (jonas2808) [inline sth_app/lib/pages/chatscreen/chatscreen.dart:4]: ChatScreen

File: sth_app/lib/pages/chatscreen/chatscreen.dart [hunk 4]
```diff
@@ -1 +1,48 @@
...
+
+  @override
+  _chatScreenState createState() => _chatScreenState();
+}
+
+class _chatScreenState extends State<chatScreen> {
```
Comment (jonas2808) [inline sth_app/lib/pages/chatscreen/chatscreen.dart:10]: ChatScreen

File: sth_app/lib/pages/chatscreen/chatscreen.dart [hunk 5]
```diff
@@ -1 +1,48 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('STH App'),
+        actions: <Widget>[
+          IconButton(
+            icon: const Icon(Icons.chat_bubble_outline),
+            onPressed: () {
+              Navigator.pushNamed(context, '/chatScreen');
```
Comment (jonas2808) [inline sth_app/lib/pages/chatscreen/chatscreen.dart:20]: /chatscreen

File: sth_app/lib/main.dart (hunk 1 above)
Comment (DevHasan7) [reply sth_app/lib/main.dart]: yes

**Review by WladimirDiCaprio — APPROVED** [review]

**Review by WladimirDiCaprio — APPROVED** [review]

**Review by jonas2808 — APPROVED** [review]

**Review by Vatsegkan — APPROVED** [review]

**Pull request merged**